### DISCUSSION TITLE
Error Wifi Digital Meter PJ-1103 Power Meter [Help]

### DISCUSSION THREAD

**Discussion by Casadomoticap** [make-all/tuya-local#553] [Q&A]
Hi everyone, 

I had been using for many weeks Tuya Wifi Digital Meter PJ-1103 Power Meter and suddenly (since 2023/03/14) it stoped to work with this local integration, I don't know why. 

This is the diagnostic but I don't understand it: 
[tuya_local-6a9c1a2ee6b46b39e0222183f15dac48-Clamp power meter-4d506800f76eccbf46372d6a603faee0.json.txt](https://github.com/make-all/tuya-local/files/11010804/tuya_local-6a9c1a2ee6b46b39e0222183f15dac48-Clamp.power.meter-4d506800f76eccbf46372d6a603faee0.json.txt)

Now the entity show "Not available" in Home Assitant but in IoT Tuya it works, the status is online and the DPs reports fine in IoT Tuya. I checked the Service API in IoT Tuya and all are "in service" 

![Captura de pantalla 2023-03-19 a las 12 39 22](https://user-images.githubusercontent.com/118067175/226172925-0f3640d0-198d-4b49-b51a-e79bae94417a.png)

![Captura de pantalla 2023-03-19 a las 12 41 01](https://user-images.githubusercontent.com/118067175/226172931-9d4b2823-6105-48e6-8db3-882d70403d25.png)

Someone have the same problem? or know how to solve it?

**Comment by make-all** [comment]
The diagnostics just confirms that the device has not connected successfully since HA was last restarted.  There might be something in the HA logs that provides more information about why.

Did you do any updates on March 14?

**Reply by Casadomoticap** [reply]
The LOGs only say:

Este error se originó a partir de una integración personalizada.

Logger: custom_components.tuya_local.device
Source: custom_components/tuya_local/device.py:184
Integration: Tuya Local (documentation, issues)
First occurred: 12:46:09 (219 occurrences)
Last logged: 13:25:10

Failed to refresh device state for Clamp power meter

I did updates on March 9 and March 15, but they don't match with the time of the error (14 March 2023, 06:14:12
)

I restored a back up from March 9 when It works and it does not work with the previous configuration either.

**Comment by make-all** [comment]
I suggest you go through the comments on #522, as there are a lot of hints in there about overcoming connection issues.
